src/benchmarks/micro/coreclr/Math/Functions/Double/AcoshDouble.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace Functions
+{
+    [BenchmarkCategory(Categories.CoreCLR)]
+    public class AcoshDouble
```

Comment:
The naming convention here should be:
class name: `public partial class MathTests`
benchmark method name: `AcoshDoubleBenchmark`

for reference please take a look at [AcosDouble.cs](https://github.com/dotnet/performance/blob/master/src/benchmarks/micro/coreclr/Math/Functions/Double/AcosDouble.cs)

Reply:
How would I modify `IterationSetup` to only apply to a given test (given that we need a different one for each `[Benchmark]` to generate the input value from the given domain)?

Reply:
@tannergooding Setup/Clean attributes have `Target` property which allows specifying target benchmark for each Setup/Cleanup methods. Thus, you can write separate IterationSetup methods for each benchmark (a usage example: https://benchmarkdotnet.org/articles/samples/IntroSetupCleanupTarget.html)